src/core/qgsdatasourceuri.h

```diff
@@ -47,10 +47,10 @@ class CORE_EXPORT QgsDataSourceURI
     QgsDataSourceURI( const QByteArray & uri );
 
     //! return connection part of URI
-    QString connectionInfo() const;
+    QString connectionInfo( bool expandAuthConfig = true ) const;
```

Comment:
can you add @param tags for expandAuthConfig? It's probably not obvious what this parameter does if you don't have knowledge of how the auth system works